tests/test_commands.py

```diff
@@ -1,60 +1,53 @@
-import copy
+import datetime
 
 from django.core.management import call_command
 from django.test import TestCase
 
+from django_ses.management.commands import get_ses_statistics as mod_get_ses_statistics
 from django_ses.models import SESStat
 
-from boto.ses import SESConnection
-
-
 data_points = [
     {
         'Complaints': '1',
-        'Timestamp': '2012-01-01T02:00:00Z',
+        'Timestamp': datetime.datetime(year=2012, month=1, day=1, hour=2),
```

Comment:
@GitRon @trecouvr has anyone tested this manually? Just want to be sure it at least works to run the stats and the returned data has this shape still... I'd assume so, but it would likely come up as a regression if not.

Reply:
About 2 weeks ago I tested the statistics command and realised that it now returns a date object rather than a string. So it should work like this 😅